src/mame/heathkit/h89.cpp

```diff
@@ -163,23 +297,118 @@ static INPUT_PORTS_START( h89 )
 INPUT_PORTS_END
 
 
+void h89_state::machine_start()
+{
+	save_item(NAME(m_gpp));
+	save_item(NAME(m_rom_enabled));
+	save_item(NAME(m_timer_intr_enabled));
+	save_item(NAME(m_floppy_ram_wp));
+
+
+	u8 *m_floppy_ram_ptr = m_floppy_ram->pointer();
+
+	// Floppy RAM - Read/Only. note: it's not rom, but behaves like it in this
+	// view.
+	m_mem_view[0].install_rom(0x1400, 0x17ff, m_floppy_ram_ptr);
+
+	// Floppy RAM - Read/write
+	m_mem_view[1].install_ram(0x1400, 0x17ff, m_floppy_ram_ptr);
```

Comment:
As noted above, it’s better to use a memory share for this and install the read side directly in the memory map.